scripts/sync

```diff
@@ -26,40 +26,47 @@ main() {
 
     info 'Generating a list of crawled urls.'
     all_crawled_packages="$(./scripts/crawled-inventory "$package_list_dir")"
-    echo "$all_crawled_packages"
-    echo
+    echo "$all_crawled_packages" > all-crawled-packages.log
+    echo "Found $(wc -l all-crawled-packages.log) crawled packages"
 
     info 'Generating a list of bucket files.'
     all_bucket_files="$(./scripts/package-inventory "$bucket_name_list")"
-    echo "$all_bucket_files"
-    echo
+    echo "$all_bucket_files" > all-bucket-files.log
+    echo "Found $(wc -l all-bucket-files.log) bucket files"
 
     info 'Downloading missing files from origin source.'
     while read -r url; do
         # Simplify url into the naming scheme used in the bucket.
         filename="$(simplify "$url")"
 
         # Check if this specific file exists in the bucket listing.
-        if ! echo "$all_bucket_files" | contains_exact "$filename"; then
-            echo "Downloading $url"
+        if ! echo "$all_bucket_files" 2>/dev/null | contains_exact "$filename"; then
```

Comment:
Why is this redirect needed?

Reply:
I'm not sure that it is... I'll remove it.